components/image-builder-api/imgbuilder.proto

```diff
@@ -34,11 +34,13 @@ message BuildSourceReference {
     string ref = 1;
 }
 
+
 message BuildSourceDockerfile {
     contentservice.WorkspaceInitializer source = 1;
     string dockerfile_version = 2;
     string dockerfile_path = 3;
     string context_path = 4;
+    string dockerfile_content_sha = 5;
```

Comment:
We could re-use the `dockerfile_version` - I don't think we need to change the protocol here.
The only difference is that before the version was the commit hash, now it's the SHA hash.

If we moved the "legacy" behaviour to server, that is.